tests/gen/gen_auto.go

```diff
@@ -212,13 +212,13 @@ func validateDefinitions(definitions []client.CollectionDefinition) error {
 	colNames := make(map[string]struct{})
 	fieldRefs := []string{}
 	for _, def := range definitions {
-		if def.Description.Name == "" {
+		if def.Description.Name.Value() == "" {
```

Comment:
> question: Why have you skipped `HasValue()` here before calling `Value()`. If IRRC the value returned is invalid unless `HasValue() == true`

question: Same as above, is interesting here because it might be the case that the Value == true & "" or == false and default (""). Even though you might want to return Error in Both case.

and other places in the file

Reply:
This was already a very strange check to have IMO. It will always have a non "" value